packages/tradershub/src/features/cfd/modals/MT5PasswordModal/MT5PasswordModal.tsx

```diff
@@ -0,0 +1,304 @@
+                    createMT5AccountLoading ||
+                    tradingPlatformPasswordChangeLoading ||
+                    !validPassword(password)
+                }
+                isLoading={tradingPlatformPasswordChangeLoading || createMT5AccountLoading}
+                onClick={onSubmit}
+                size='lg'
+            >
+                Create Deriv MT5 password
+            </Button>
+        );
+    }, [
+        createMT5AccountLoading,
+        hasMT5Account,
+        isSuccess,
+        onSubmit,
+        password,
+        renderSuccessButton,
+        show,
+        tradingPlatformPasswordChangeLoading,
+    ]);
+
+    const successComponent = useMemo(() => {
+        const renderSuccessDescription = () => {
+            return isDemo
+                ? `Let's practise trading with ${ActiveTradingAccount?.display_balance} virtual funds.`
+                : `Transfer funds from your ${ActiveTradingAccount?.currency} Wallet to your ${marketTypeTitle} ${landingCompanyName} account to start trading.`;
+        };
+
+        if (isSuccess) {
+            return (
+                <CFDSuccess
+                    description={renderSuccessDescription()}
+                    displayBalance={
+                        mt5Accounts?.find(account => account.market_type === marketType)?.display_balance ?? '0.00'
+                    }
+                    landingCompany={selectedJurisdiction}
+                    marketType={marketType}
+                    platform='mt5'
+                    renderButtons={renderSuccessButton}
+                    title={`Your ${marketTypeTitle} ${isDemo ? ' demo' : landingCompanyName} account is ready`}
+                />
+            );
+        }
+    }, [
+        isSuccess,
+        isDemo,
+        ActiveTradingAccount?.currency,
+        ActiveTradingAccount?.display_balance,
+        marketTypeTitle,
+        landingCompanyName,
+        mt5Accounts,
+        selectedJurisdiction,
+        marketType,
+        renderSuccessButton,
+    ]);
+
+    const passwordComponent = useMemo(() => {
+        if (!isSuccess) {
+            return isMT5PasswordNotSet ? (
+                <CreatePassword
+                    icon={<MT5PasswordIcon />}
+                    isLoading={tradingPlatformPasswordChangeLoading || createMT5AccountLoading}
+                    onPasswordChange={e => setPassword(e.target.value)}
+                    onPrimaryClick={onSubmit}
+                    password={password}
+                    platform={PlatformDetails.mt5.platform}
+                />
+            ) : (
+                <EnterPassword
+                    isLoading={tradingPlatformPasswordChangeLoading || createMT5AccountLoading}
+                    marketType={marketType}
+                    onPasswordChange={e => setPassword(e.target.value)}
+                    onPrimaryClick={onSubmit}
+                    // onSecondaryClick={() => <show SentEmailContent platform={platform} />}
```

Comment:
can uncomment this. SentEmailContent is already merged

Reply:
awesome!